exampleCourse/courseInstances/SectionA/assessments/00-QuestionGallery/infoAssessment.json

```
@@ -0,0 +1,35 @@
+{
+    "uuid": "13D012E5-9C66-4092-BA1A-0445919CB0C3",
+    "type": "Homework",
+    "title": "Questions from the gallery at https://www.prairielearn.com",
+    "set": "Collection",
+    "number": "1",
+      "comment": "you must add access rules below to make this assessment available to students",
```

I'm increasingly thinking that we should be using Prettier here!
```suggestion
    "comment": "you must add access rules below to make this assessment available to students",
```

---

For sure!